p2p/src/net/default_backend/backend.rs

```diff
@@ -57,6 +57,10 @@ use crate::{
 
 use super::{peer::PeerRole, transport::TransportAddress, types::HandshakeNonce};
 
+/// Buffer size of the channel to the SyncManager peer task.
+/// How many unprocessed messages can be sent before the peer's event loop is blocked.
+const SYNC_CHAN_BUF_SIZE: usize = 20;
```

Comment:
This sounds too small... are you sure this is an appropriate value? Not like 100-500?

Reply:
It's per connected peer, so 20 should be enough.
I guess it should mostly affect how much time is spent on initial blocks download, so I tried to test how much time it takes to synchronize a new node. The result is surprising:
```
Unbounded channels (current master): 4:36.10
Bounded channels (buffer is 20): 2:46.00
Bounded channels (buffer is 500): 4:18.96
```
I noticed that a lot of time is spent in mempool, which is probably pointless if the node is in initial block download state. I will create a ticket for this. We could test again once the issue with mempool is resolved (I added TODO).

Reply:
Small update. I think I now understand why the times above are so different. I was stopping the timer when the node appeared to be in sync by stopping printing to the console, and a lot of time was actually spent printing from mempool. I got these times by checking the chainstate RPC instead:
```
Unbounded (current master): 1:29.46
Bounded (20): 1:31.60
Bounded (100): 1:28.44
Bounded (500) 1:30.68
```
So the buffer size of 20 should be to be sufficient.